### PR TITLE
remove deprecated functions

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- title

### CODE REVIEW

**Comment by commy2** [comment]
Bye bye `filter`, `map` and `execRemoteFnc`.

2014-2016
https://github.com/KoffeinFlummi/AGM/commit/bcc79f026ffbc95ca3eacd1e81f53f4367a828cd
https://github.com/KoffeinFlummi/AGM/commit/3a36ca47f88c7c68c241d382e81fef8b86f4811b

File: addons/common/XEH_postInit.sqf [hunk 1]
```diff
@@ -326,7 +318,37 @@ enableCamShake true;
...
+            primaryWeapon _this, primaryWeaponItems _this, primaryWeaponMagazine _this,
+            secondaryWeapon _this, secondaryWeaponItems _this, secondaryWeaponMagazine _this,
+            handgunWeapon _this, handgunItems _this, handgunMagazine _this,
+            assignedItems _this,
+            binocular _this,
+            _this call CBA_fnc_binocularMagazine
+        ]
+    };
+
+    ["ace_playerInventoryChanged", [ACE_player, ACE_player call _fnc_getAllGear]] call CBA_fnc_localEvent;
```
Comment (SilentSpike) [inline addons/common/XEH_postInit.sqf]: Missing `{}` ?

Comment (esteldunedain) [inline addons/common/XEH_postInit.sqf]: aye

Comment (commy2) [inline addons/common/XEH_postInit.sqf]: No.

Comment (commy2) [inline addons/common/XEH_postInit.sqf]: `CBA_fnc_localEvent` != `CBA_fnc_addEventHandler`

I want to pass the return value of `_fnc_getAllGear` and not some \<CODE\>

Comment (esteldunedain) [inline addons/common/XEH_postInit.sqf]: aye, right

**Comment by SilentSpike** [comment]
:laughing: Ah yep, totally overlooked that

File: addons/common/XEH_postInit.sqf (hunk 1 above)
Comment (SilentSpike) [inline addons/common/XEH_postInit.sqf]: :laughing: Ah yep, totally overlooked that

**Pull request merged**